devops/pipeline/binary_package.yaml

```diff
@@ -8,119 +8,116 @@ name: $(MajorVersion).$(MinorVersion).$(PatchVersion).$(Date:yyyyMMdd)$(Rev:rr)
         platform: linux/arm/v7
-        performCoverage: false
-        ignoreTestFailures: false
-        publishArtifact: true
-      - variantName: ubuntu20.04
-        friendlyVariantName: ubuntu2004
+        displayName: debian9_arm
+
+      - os: debian9
         architecture: arm64
-        containerPath: $(CONTAINER_REGISTRY)/ubuntu20.04-dev-arm64:$(CONTAINER_TAG)
         platform: linux/arm64/v8
-        performCoverage: false
-        ignoreTestFailures: false
-        publishArtifact: true
-      - variantName: ubuntu18.04
-        friendlyVariantName: ubuntu1804
+        displayName: debian9_arm64
+
+      - os: debian10
         architecture: amd64
-        containerPath: $(CONTAINER_REGISTRY)/ubuntu18.04-dev-amd64:$(CONTAINER_TAG)
         platform: linux/amd64
-        performCoverage: false
-        ignoreTestFailures: false
-        publishArtifact: true
-      - variantName: ubuntu18.04
-        friendlyVariantName: ubuntu1804
+        displayName: debian10_amd64
+
+      - os: debian10
         architecture: arm
-        containerPath: $(CONTAINER_REGISTRY)/ubuntu18.04-dev-arm:$(CONTAINER_TAG)
         platform: linux/arm/v7
-        performCoverage: false
-        ignoreTestFailures: false
-        publishArtifact: true
-      - variantName: ubuntu18.04
-        friendlyVariantName: ubuntu1804
+        displayName: debian10_arm
+
+      - os: debian10
         architecture: arm64
-        containerPath: $(CONTAINER_REGISTRY)/ubuntu18.04-dev-arm64:$(CONTAINER_TAG)
         platform: linux/arm64/v8
-        performCoverage: false
-        ignoreTestFailures: false
-        publishArtifact: true
-      - variantName: debian9
-        friendlyVariantName: debian9
+        displayName: debian10_arm64
+
+      - os: debian11
         architecture: amd64
-        containerPath: $(CONTAINER_REGISTRY)/debian9-dev-amd64:$(CONTAINER_TAG)
         platform: linux/amd64
-        performCoverage: false
-        ignoreTestFailures: false
-        publishArtifact: true
-      - variantName: debian9
-        friendlyVariantName: debian9
+        displayName: debian11_amd64
+
+      - os: debian11
         architecture: arm
-        containerPath: $(CONTAINER_REGISTRY)/debian9-dev-arm:$(CONTAINER_TAG)
         platform: linux/arm/v7
-        performCoverage: false
-        ignoreTestFailures: false
-        publishArtifact: true
-      - variantName: debian9
-        friendlyVariantName: debian9
+        displayName: debian11_arm
+
+      - os: debian11
         architecture: arm64
-        containerPath: $(CONTAINER_REGISTRY)/debian9-dev-arm64:$(CONTAINER_TAG)
         platform: linux/arm64/v8
-        performCoverage: false
-        ignoreTestFailures: false
-        publishArtifact: true
+        displayName: debian11_arm64
 
 variables:
   SERVICE_CONNECTION: OSConfig-ACR
   CONTAINER_REGISTRY: osconfig.azurecr.io
-  CONTAINER_TAG: 289651
+  CONTAINER_TAG: latest
```

Comment:
We should stick to a specific tag and not always use `latest`

Reply:
Can you please elaborate why ("we should stick to a specific tag")?

Reply:
The issue with a "specific" tag is that it required us to build all (now 15) containers every time we need to make an update to any of the Dockerfiles. Latest will also let us "roll-back" if there is an issue with a new image that is built.

Reply:
> Can you please elaborate why ("we should stick to a specific tag")?

This is to have explicit control over the container version. Although this was more of a problem in the past where containers were updated without our CD workflows. Now this is less of a problem given we have CD for the creating the containers we just automagically pick up the latest.

> The issue with a "specific" tag is that it required us to build all (now 15) containers every time we need to make an update to any of the Dockerfiles. Latest will also let us "roll-back" if there is an issue with a new image that is built.

Yeah i get the convenience. This was more of an issue in the past.